packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'framework.dart';
+
+/// Embeds an Android view in the Widget hierarchy.
+///
+/// Embedding Android views is an expensive operation and should be avoided when a Flutter
```

Comment:
Should we explain that an embedded android view behaves just like a normal Flutter widget with respect to painting and input. And what about layout? I imagine that we need to explain that the parent has to constrain the size of its AndroidView.

Reply:
Added:
```dart
///  The embedded Android view is painted just like any other Flutter widget and transformations
/// apply to it as well.
```
Anything else to add here?

I'll leave the comment about input to the PR where I add input support.

Reply:
Looks good